src/cltk/alphabet/lat.py

```diff
@@ -1,12 +1,471 @@
+- If only the end of the line is missing, they use a single bracket ʺ[...ʺ
+- If the lineʹs beginning is missing, they use ʺ...]ʺ
+- Within the brackets, often each dot represents one missing letter.
+- [[...]] Double brackets enclose letters or words deleted by the medieval copyist himself.
+- (...) Round brackets are used to supplement words abbreviated by the original copyist; e.g. in an inscription: ʺtrib(unus) mil(itum) leg(ionis) IIIʺ
+- <...> diamond ( = elbow = angular) brackets enclose words etc. that an editor has added (see ʺsuppl.ʺ)
+- †  An obelus (pl. obeli) means that the word(s etc.) is very plainly corrrupt, but the editor cannot see how to emend.
+- If only one word is corrupt, there is only one obelus, which precedes the word; if two or more words are corrupt, two obeli enclose them. (Such at least is the rule--but that rule is often broken, especially in older editions, which sometimes dagger several words using only one obelus.) To dagger words in this way is to ʺobelizeʺ them.
+
+"""
+__author__ = [
+    "Todd Cook <todd.g.cook@gmail.com>",
+    "Kyle P. Johnson <kyle@kyle-p-johnson.com>",
+]
+__license__ = "MIT License"
+
+import re
+from typing import Dict
 
 from cltk.alphabet.text_normalization import cltk_normalize
 
+BRACE_STRIP = re.compile(r"{[^}]+}")
+NUMERALS = re.compile(r"[0-9]+")
+LATIN_PUNCT = re.compile(r"[\\/':;,!\?\._『@#\$%^&\*]+")
+QUOTES = re.compile(r'["”“]+')
+# we will extact content between brackets, it is editorial
+ANGLE_BRACKETS = re.compile(r"([a-zA-Z]+)?<[a-zA-Z\s]+>([,\?\.a-zA-Z]+)?")
+SQUARE_BRACKETS = re.compile(r"\[[^\]]+\]")
+OBELIZED_WORDS = re.compile(r"†[^†]+†")
+OBELIZED_WORD = re.compile(r"†[^\s]+\s")
+OBELIZED_PLUS_WORDS = re.compile(r"[\+][^\+]+[\+]")
+OBELIZED_PLUS_WORD = re.compile(r"[\+][^\s]+\s")
+HYPHENATED = re.compile(r"\s[^-]+-[^-]+\s")
+
+
+class JVReplacer:  # pylint: disable=too-few-public-methods
+    """Replace J/V with I/U.
+    Latin alphabet does not distinguish between J/j and I/i and V/v and U/u;
+    Yet, many texts bear the influence of later editors and the predilections of other languages.
+
+    In practical terms, the JV substitution is recommended on all Latin text preprocessing; it
+    helps to collapse the search space.
+
+    >>> replacer = JVReplacer()
+    >>> replacer.replace("Julius Caesar")
+    'Iulius Caesar'
+
+    >>> replacer.replace("In vino veritas.")
+    'In uino ueritas.'
+
+    """
+
+    def __init__(self):
+        """Initialization for JVReplacer, reads replacement pattern tuple."""
+        patterns = [(r"j", "i"), (r"v", "u"), (r"J", "I"), (r"V", "U")]
+        self.patterns = [(re.compile(regex), repl) for (regex, repl) in patterns]
+
+    def replace(self, text):
+        """Do j/v replacement"""
+        for (pattern, repl) in self.patterns:
+            text = re.subn(pattern, repl, text)[0]
+        return text
+
+
+jvreplacer = JVReplacer()
+
+
+class AEOEReplacer:  # pylint: disable=too-few-public-methods
```

Comment:
See note below, consider calling it `LigatureReplacer`. If uncommon Medieval Latin ligatures ever cropped up in a text, we could deal with them here. All possibilities, though granted none are common: https://en.wikipedia.org/wiki/Orthographic_ligature#Latin_alphabet

Reply:
Good point, renamed in all instances.

Reply:
> jvreplacer = JVReplacer()
> aeoereplacer = AEOEReplacer()

I haven't thought enough about this, but would like to make this another nit-picky thing to reconsider in the future.